superbuild/15_0_apple_disable_ios.patch

```diff
@@ -0,0 +1,26 @@
+From a49d5f847d169fd79bf0f528dd8d4681089d2dbb Mon Sep 17 00:00:00 2001
+From: Alexey Nurmukhamedov <anurmukh@sctpimac1.sc.intel.com>
+Date: Fri, 14 Jul 2023 08:34:42 -0700
+Subject: [PATCH] Fix compiler-rt build issue
+
+see issue https://github.com/llvm/llvm-project/issues/57037
```

Comment:
Do we need a patch for that? Can this be achieved by just passing `COMPILER_RT_ENABLE_IOS` value passed externally?

Reply:
I need to check that.

Reply:
It looks like it works. I have been misled by LLVM ticket and tried `-DBUILTINS_CMAKE_ARGS=-DCOMPILER_RT_ENABLE_IOS=OFF` that didn't work for me.